pkg/ruler/ruler.go

```diff
@@ -132,6 +136,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.DurationVar(&cfg.FlushCheckPeriod, "ruler.flush-period", 1*time.Minute, "Period with which to attempt to flush rule groups.")
 	f.StringVar(&cfg.RulePath, "ruler.rule-path", "/rules", "file path to store temporary rule files for the prometheus rule managers")
 	f.BoolVar(&cfg.EnableAPI, "experimental.ruler.enable-api", false, "Enable the ruler api")
+	f.DurationVar(&cfg.OutageTolerance, "ruler.outage-tolerance", 0, "outage period after which previous alert state cannot be cannot be checked from metric storage and must be re-evaluated in full.")
```

Comment:
Can you update this flag to look more like:

https://github.com/prometheus/prometheus/blob/b788986717e1597452ca25e5219510bb787165c7/cmd/prometheus/main.go#L231-L232

Keep the `ruler.` prefix but in general it would be good to keep a similar naming scheme